PSScriptAnalyzerRules.psd1

```diff
@@ -0,0 +1,84 @@
+@{
+    Severity     = @('Error')
+    IncludeRules = @(
+        'PSUseCompatibleSyntax',
+        'PSAvoidUsingCmdletAliases',
+        'PSAvoidUsingWriteHost',
+        'PSAvoidDefaultValueSwitchParameter',
+        'PSReservedCmdletChar',
+        'PSReservedParams',
+        'PSAvoidUsingUserNameAndPassWordParams',
+        'PSAvoidUsingPlaintTextForPassword',
+        'PSAvoidUsingWMICmdlet',
+        'PSAvoidUsingWriteHost',
+        'PSMisleadingBacktick',
+        'PSMissingModuleManifestField',
+        'PSPossibleIncorrectComparisonWithNull',
+        'PSUseApprovedVerbs',
+        'PSUseOutputTypeCorrectly',
+        'PSShouldProcess',
+        'PSUserToExportFieldsInManifest',
+        'PSUseSingularNouns',
+        'PSAvoidUsingInvokeExpression',
+        'PSUseShouldProcessForStateChangingFunctions',
+        'PSUseDeclaredVarsMoreThanAssignments',
+        'PSUseCore',
```

Comment:
Please explain what the `PSUseCore` is. I have searched in the PSScriptAnalyzer repository but could not find a rule with this name.

Reply:
Same here.

```suggestion
```